examples/blog-refine-nextui/src/pages/products/list.tsx

```diff
@@ -0,0 +1,303 @@
+      <Table
+        isStriped
+        aria-label="Products table"
+        sortDescriptor={sortDescriptor}
+        onSortChange={(e) => {
+          const sorter: CrudSort = {
+            order: e.direction === "ascending" ? "asc" : "desc",
+            field: e.column as string,
+          };
+
+          if (e.column === "category") {
+            sorter.field = "category.title";
+          }
+
+          setSorters([sorter]);
+          setSortDescriptor(e);
+        }}
+        topContent={
+          <div className="flex flex-col gap-4">
+            <div className="flex justify-between gap-3">
+              <h1 className="font-bold">Products</h1>
+              <Button
+                color="primary"
+                variant="bordered"
+                startContent={<PlusIcon className="h-4 w-4" />}
+                onPress={() => {
+                  create("products");
+                }}
+              >
+                Create Product
+              </Button>
+            </div>
+            <div className="flex justify-end">
+              <Input
+                isClearable
+                className="w-full sm:max-w-[20%]"
+                placeholder="Search"
+                startContent={<MagnifyingGlassIcon width={12} />}
+                value={getDefaultFilter("q", filters)}
+                onClear={() => {
+                  setCurrent(1);
+                  setFilters([], "replace");
+                }}
+                onValueChange={(value) => {
+                  if (!value.trim()) {
+                    setCurrent(1);
+                    setFilters([], "replace");
+                    return;
+                  }
+
+                  setCurrent(1);
+                  setFilters([
+                    {
+                      field: "q",
+                      value: value,
+                      operator: "contains",
+                    },
+                  ]);
+                }}
+              />
+            </div>
+          </div>
+        }
+        bottomContent={
+          <div className="flex w-full gap-2 justify-center">
+            <Pagination
+              isCompact
+              showControls
+              showShadow
+              color="primary"
+              page={current}
+              total={pageCount}
+              onChange={(page) => setCurrent(page)}
+            />
+            <Dropdown>
+              <DropdownTrigger>
+                <Button variant="bordered" className="capitalize">
+                  Rows per page
+                </Button>
+              </DropdownTrigger>
+              <DropdownMenu
+                aria-label="Select number of rows per page"
+                variant="flat"
+                color="primary"
+                disallowEmptySelection
+                selectionMode="single"
+                selectedKeys={[`${pageSize}`]}
+                onSelectionChange={(rows) => {
+                  console.log(+(rows as Set<string>).values().next().value);
```

Comment:
Is this left on purpose?
```suggestion
                  console.log(+(rows as Set<string>).values().next().value);
```